### PR TITLE
@orta => Return value from resolver

### PR DESCRIPTION
**Pull request by mzikherman**
This is a 1-liner fix, but something still keeps getting awry with my local setup, I always have these diffs (cc @damassi who tried to help, we thought it was just when I was touching files that weren't touched in a while), and this is a new file I believe.

### CODE REVIEW

File: src/lib/graphqlTimeoutMiddleware.ts [hunk 1]
```diff
@@ -62,25 +72,32 @@ export const graphqlTimeoutMiddleware = (defaultTimeoutInMS: number) => {
...
+              } has timed out after waiting for ${timeoutInMS}ms`
+            )
+          )
         }, timeoutInMS)
       }),
       resolverResult.then(
         result => {
           clearTimeout(timeoutID)
-          result
+          return result
```
Comment (mzikherman) [inline src/lib/graphqlTimeoutMiddleware.ts]: This is the fix.

Comment (damassi) [reply src/lib/graphqlTimeoutMiddleware.ts]: Ah, funny that this didn't get flagged from @mbilokonsky linting work. 

As to formatting, your prettier is working now!

Comment (orta) [reply src/lib/graphqlTimeoutMiddleware.ts]: 👍

**Comment by orta** [comment]
merge on green

File: src/lib/graphqlTimeoutMiddleware.ts (hunk 1 above)
Comment (orta) [reply src/lib/graphqlTimeoutMiddleware.ts]: Nice one

File: src/lib/graphqlTimeoutMiddleware.ts [hunk 2]
```diff
@@ -15,16 +15,26 @@ export function fieldFromResolveInfo(resolveInfo: GraphQLResolveInfo) {
 
 export function timeoutForField(field: GraphQLField<any, any>) {
   const fieldDirectives = field.astNode && field.astNode.directives
-  const directive = fieldDirectives && fieldDirectives.find(directive => directive.name.value === "timeout")
```
Comment (damassi) [inline src/lib/graphqlTimeoutMiddleware.ts]: @mzikherman - if this is a new file then something wasn't set up -- this is far wider than 80 characters.

Comment (orta) [reply src/lib/graphqlTimeoutMiddleware.ts]: It's a typescript file, so maybe it wasn't getting hit by some of the pre-commit hooks?

File: src/lib/graphqlTimeoutMiddleware.ts (hunk 1 above)
Comment (alloy) [reply src/lib/graphqlTimeoutMiddleware.ts]: Oh man, I don’t get how there wasn’t any tests failing for this 🤔 Did you take a look at why they have been insufficient?

File: src/lib/graphqlTimeoutMiddleware.ts (hunk 2 above)
Comment (damassi) [reply src/lib/graphqlTimeoutMiddleware.ts]: @orta - yup thats it: https://github.com/artsy/metaphysics/blob/master/package.json#L171

File: src/lib/graphqlTimeoutMiddleware.ts (hunk 1 above)
Comment (orta) [reply src/lib/graphqlTimeoutMiddleware.ts]: likely that the queries running in unit tests don't have the middleware applied

Comment (orta) [reply src/lib/graphqlTimeoutMiddleware.ts]: I'm gonna try write a single integration test tomorrow on metaphysics to catch this kind of thing

**Pull request merged**